<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recorridos</class>
 <widget class="QWidget" name="Recorridos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lHoraSalida">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HORA DE SALIDA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lIDConductor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID CONDUCTOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPasajeros">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PASAJEROS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbEstLlegada">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PISCINA OLIMPICA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ESPOL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACACIAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TERMINAL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ORQUIDEAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DURAN</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lFecha">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FECHA</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lEstSalida">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ESTACION SALIDA</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditHoraLLegada">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lEstLlegada">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ESTACION LLEGADA</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleRecorridos">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>40</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RECORRIDOS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbEstSalida">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PISCINA OLIMPICA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ESPOL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACACIAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TERMINAL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ORQUIDEAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DURAN</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEPasajeros">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditHoraSalida">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEIDConductor">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lHoraLLegada">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HORA DE LLEGADA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lIDUnidad">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID UNIDAD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEIDUnidad">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bRegresarRecorridos">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Descargas/IMAGENES/Regresar.jpg</normaloff>../../Descargas/IMAGENES/Regresar.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>1000</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bingresarRecorridos">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Descargas/IMAGENES/Ingresar.jpg</normaloff>../../Descargas/IMAGENES/Ingresar.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>600</width>
     <height>360</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
